src/services/messagesService.ts

```diff
@@ -49,7 +55,9 @@ export default class MessagesService {
             ? ResponseSuccessJson(response.value)
             : response.status === 404
               ? ResponseErrorNotFound("Not found", "User not found")
-              : unhandledResponseStatus(response.status)
+              : response.status === 429
+                ? ResponseErrorTooManyRequests(this.magTooManyRequests)
```

Comment:
you may want to use something else for the detail, "Too many requests" is already [in the title of the response](https://github.com/teamdigitale/io-ts-commons/blob/master/src/responses.ts#L388)

Reply:
For example: Too many request response from upstream API

Reply:
yes that would be awesome - in that case you need to add the proper response type to the functions specs - the 429 comes from [Azure Functions runtime](https://github.com/Azure/azure-functions-host/wiki/Http-Functions#throttling) or [Azure APIM](https://docs.microsoft.com/en-us/azure/api-management/transform-api) - not sure if there's a standard payload that works for both.

Another option would be to make the `detail` param in `ResponseErrorTooManyRequests` optional instead of required.

Reply:
Ok I make the detail param as optional.